charts/kong/values.yaml

```
@@ -78,10 +78,11 @@ admin:
   # Enterprise users that wish to use Kong Manager with the controller should enable this
   enabled: false
   type: NodePort
-  # If you want to specify annotations for the admin service, uncomment the following
-  # line, add additional or adjust as needed, and remove the curly braces after 'annotations:'.
+  # To specify annotations or labels for the admin service, remove the "{}" and add indented lines
+  # after, similar to the commented aws-load-balancer-proxy-protocol example
```

```suggestion
  # To specify annotations or labels for the admin service, add them to the respective
  # "annotations" or "labels" dictionaries below.
```

Instead of asking users to manipulate opaque strings, we should be speaking about YAML concepts, right?

Applies to all repetitions.